src/main/java/uk/org/tombolo/ImporterSpecificationRunner.java

```diff
@@ -0,0 +1,120 @@
+package uk.org.tombolo;
+
+import org.apache.commons.lang.exception.ExceptionUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.org.tombolo.core.Attribute;
+import uk.org.tombolo.core.Datasource;
+import uk.org.tombolo.core.SubjectType;
+import uk.org.tombolo.importer.Importer;
+
+import java.lang.reflect.Constructor;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Get a list of datasources for an importer
+ */
+public class ImporterSpecificationRunner extends AbstractRunner {
+    private static final Logger log = LoggerFactory.getLogger(ImporterSpecificationRunner.class);
+    private static final ImporterSpecificationRunner runner = new ImporterSpecificationRunner();
+
+    public static void main(String[] args) throws Exception {
+
+        boolean subjects = Boolean.parseBoolean(args[1]);
+        boolean attribute = Boolean.parseBoolean(args[2]);
+        boolean provider = Boolean.parseBoolean(args[3]);
+        String importerClass = args[0];
+        StringBuilder info = new StringBuilder("\n\n");
+
+        if (importerClass.equals("None")) {
+            CatalogueExportRunner cer = new CatalogueExportRunner();
+            List<Class<? extends Importer>> dcImporters = cer.getImporterClasses();
```

Comment:
Can we add a link to the [importers in the documentation](https://github.com/FutureCitiesCatapult/TomboloDigitalConnector/blob/master/documentation/importers.md)?